aerospike_helpers/metrics/__init__.py

```diff
@@ -38,6 +41,13 @@ class ConnectionStats:
     pass
 
 
+ERROR_COUNT_DOCSTRING = "Command error count since node was initialized. If the error is retryable, multiple errors \
+    per command may occur."
+TIMEOUT_COUNT_DOCSTRING = "Command timeout count since node was initialized. If the timeout is retryable \
+    (i.e socket_timeout), multiple timeouts per command may occur."
+KEY_BUSY_COUNT_DOCSTRING = "Command key busy error count since node was initialized."
```

Comment:
Maybe mark these as private with a leading "_"? I don't think these need to be exposed. What do you think?

Reply:
I agree, I forgot that people can import these constants in their code if I don't make them private